openaz-pep/src/main/java/org/apache/openaz/pepapi/std/ActionMapper.java

```diff
@@ -33,14 +32,14 @@ public ActionMapper() {
 
     @Override
     public void map(Object o, PepRequest pepRequest) {
-        Action a = (Action) o;
-        String id = a.getId();
+        Action action = (Action) o;
+        String id = action.getId();
         if (id == null) {
             id = getPepConfig().getDefaultActionId();
```

Comment:
yes, when I changed this code, given the lack of documentation, my impression was that  `getPepConfig().getDefaultActionId()` returns a default action id value and not the action id key (which IMO would make more sense).